### ISSUE TITLE
LUA Error on "Quick Order" 

### ISSUE THREAD

**Issue by fubaWoW** [Sluimerstand/ProfessionShoppingList#96]
hi there, 

Added a "Quick Order" from my, lets call it "Main Leatherworker" for [Trailblazer's Scale Bracers](https://www.wowhead.com/item=193393) to one of my Twinks (also Leatherworker and same Server) and got the following error below every time used the "Quick Order" Button.

- Adding the order the "Regular" way -> no error.
- Adding another Orders the same way (with the Quick Order Button) like [Blazing Ink,](https://www.wowhead.com/item=194751) [Handful of Serevite Bolts](https://www.wowhead.com/item=198183) or [Wildercloth Bolt](https://www.wowhead.com/item=193926) -> no error.
- Adding the SAME Work Order from any of my NON Leatherwoker Chars -> no error.

```LUA
3x bad argument #4 to '?' (outside of expected range 0 to 4294967295 - Usage: C_CraftingOrders.PlaceNewOrder(orderInfo))
[string "=[C]"]: in function `PlaceNewOrder'
[string "@ProfessionShoppingList/ProfessionShoppingList.lua"]:785: in function <...ns/ProfessionShoppingList/ProfessionShoppingList.lua:713>

Locals:
(*temporary) = "bad argument #4 to '?' (outside of expected range 0 to 4294967295 - Usage: C_CraftingOrders.PlaceNewOrder(orderInfo))"

```

greetings

**Comment by Sluimerstand** [comment]
I've found the culprit and fixed it for v10.1.5-006. Thank you for the detailed info, it helped a lot!

**Comment by fubaWoW** [comment]
thanks for the fix! 😉
